src/components/CommentCard.jsx

```diff
@@ -1,7 +1,39 @@
 import Card from '@mui/material/Card';
 import CardContent from '@mui/material/CardContent';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+import Button from '@mui/material/Button';
+import { useEffect, useState } from 'react';
+import { deleteReviewComment, getCommentsByReviewId } from '../utils/api';
+
+const CommentCard = ({ gameComment, setGameComments, review_id, setIsLoading }) => {
+
+    const [deleteComment, setDeleteComment] = useState(0)
+    const [isError, setIsError] = useState(null)
+
+    useEffect(() => {
+      if(deleteComment !== 0){
+        setIsLoading(true)
+        deleteReviewComment(deleteComment)
+        .then((data) => {
+          getCommentsByReviewId(review_id)
```

Comment:
rather than performing another api request, you can use the setGameComments and change the state of the current comments filtering out the deleted comment